crates/bencode/Cargo.toml

```diff
@@ -10,15 +10,12 @@ readme = "README.md"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
-[features]
-default = ["sha1-system"]
-sha1-system = ["sha1w/sha1-system"]
-sha1-openssl = ["sha1w/sha1-openssl"]
-sha1-rust = ["sha1w/sha1-rust"]
-
 [dependencies]
 serde = {version = "1", features=["derive"]}
 buffers = {path = "../buffers", package="librqbit-buffers", version = "2.2.1"}
 clone_to_owned = {path = "../clone_to_owned", package="librqbit-clone-to-owned", version = "2.2.1"}
 anyhow = "1"
-sha1w = {path="../sha1w", default-features=false, package="librqbit-sha1-wrapper", version="2.2.1"}
\ No newline at end of file
+serde_bytes = "0.11.14"
```

Comment:
let's not use serde_bytes, but keep using buffers crate instead. As we have handy wrappers around [u8], Vec<u8> already, there's no need to use serde_bytes.